src/display/api.js

```diff
@@ -2125,13 +2126,24 @@ var InternalRenderTask = (function InternalRenderTaskClosure() {
     this._continueBound = this._continue.bind(this);
     this._scheduleNextBound = this._scheduleNext.bind(this);
     this._nextBound = this._next.bind(this);
+    this._canvas = params.canvasContext.canvas;
   }
 
   InternalRenderTask.prototype = {
 
     initializeGraphics:
         function InternalRenderTask_initializeGraphics(transparency) {
 
+      if (this._canvas) {
+        if (canvasInRendering.has(this._canvas)) {
+          throw new Error(
+            'Cannot use the same canvas during multiple render() operations.' +
```

Comment:
Nit: add space at the end of this string, to avoid it reading `... operations.Use different...`.